source/features/linkify-code.tsx

```diff
@@ -29,7 +29,17 @@ function linkifyContent(wrapper: Element): void {
 	// Linkify issue refs in comments, exclude gists:
 	// https://github.com/refined-github/refined-github/pull/3844#issuecomment-751427568
 	if (!pageDetect.isGist()) {
-		const currentRepo = getRepo() ?? {};
```

Comment:
Let's detect the global search page instead. When there, there's not need to call getRepo